development/nautobot_config.py

```diff
@@ -343,7 +343,7 @@
 # Django Debug Toolbar
 DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda _request: DEBUG and not TESTING}
 
-if "debug_toolbar" not in INSTALLED_APPS:  # noqa: F405
-    INSTALLED_APPS.append("debug_toolbar")  # noqa: F405
-if "debug_toolbar.middleware.DebugToolbarMiddleware" not in MIDDLEWARE:  # noqa: F405
-    MIDDLEWARE.insert(0, "debug_toolbar.middleware.DebugToolbarMiddleware")  # noqa: F405
+# if "debug_toolbar" not in INSTALLED_APPS:  # noqa: F405
```

Comment:
Maybe add a "debug" mode conditional?

Reply:
I'll remove this and let's get that as an outstanding issue since it is only for development.